config/dev/deployment.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: hmc.mirantis.com/v1alpha1
```

Comment:
Looks like you checked your `deployment.yaml` in, run a `git checkout upstream/main -- config/dev/deployment.yaml` and grab the original.